Composer/packages/lib/bot-deploy/src/botProjectDeployConfig.ts

```diff
@@ -35,7 +35,7 @@ export interface BotProjectDeployConfig {
   // The ARM template file path, default is 'DeploymentTemplates/template-with-preexisting-rg.json'
   templatePath?: string;
 
-  // Dotnet project path, default is 'BotProject.csproj'
+  // Dotnet project path, default is 'Microsoft.BotFramework.Composer.WebApp.csproj'
```

Comment:
All of these hardcoded paths are going away when we do the interchangeable runtimes DCR later on https://github.com/microsoft/BotFramework-Composer/issues/2852